pkg/flags/global.go

```diff
@@ -49,21 +50,24 @@ func (g *GlobalFlags) Register(set *cli.FlagSet) {
 	})
 
 	set.AfterParse(func(merr error) error {
+		g.FlagPlatform = strings.ToLower(strings.TrimSpace(g.FlagPlatform))
+
+		if _, ok := allowedPlatforms[g.FlagPlatform]; !ok && g.FlagPlatform != "" {
+			merr = errors.Join(merr, fmt.Errorf("unsupported value for platform flag: %s", g.FlagPlatform))
+		}
+
 		switch g.FlagPlatform {
 		case "github":
+		case "local":
 		default:
-			if os.Getenv("GITHUB_ACTIONS") == "true" {
+			if v, _ := strconv.ParseBool(set.GetEnv("GITHUB_ACTIONS")); v {
 				g.FlagPlatform = "github"
 				break
 			}
 
 			g.FlagPlatform = "local"
 		}
```

Comment:
I think you can drop all of this now:

```go
if g.FlagPlatform == "" {
	if v, _ := strconv.ParseBool(set.GetEnv("GITHUB_ACTIONS")); v {
		g.FlagPlatform = "github"
	}
}
```

Reply:
Moved to https://github.com/abcxyz/guardian/pull/367